src/main/java/com/alkemy/ong/service/impl/CategoryServiceImpl.java

```diff
@@ -44,6 +44,10 @@ public CategoryDto save(CategoryDto categoryDto) {
     @Override
     public List<CategoryNameDto> viewAllCategoryNames() {
         List<CategoryNameDto> categoryNameDtos = new ArrayList<>();
+        List<Category> lista = categoryRepository.findAll();
+        for (Category category : lista) {
```

Comment:
Por que tenes un for vacio aca? Y en caso de mantenerlo con codigo necesario a la variable lista modificale el nombre para que indique de que es la lista (y en ingles porfa)